<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recommend_music</class>
 <widget class="QWidget" name="recommend_music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1076</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	/*background-color: rgba(255, 255, 255, 0.6);*/
	background-color: transparent;
	color: #fff8ea;
	border: 2px solid white;
	font-size: 16px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}

/*垂直滚动条样式*/
QScrollBar::add-line, QScrollBar::sub-line {
    width: 0;  /* 隐藏水平滚动条的箭头 */
    height: 0; /* 隐藏垂直滚动条的箭头 */
}
QScrollBar::add-page, QScrollBar::sub-page {
    background: transparent;
    /*background: rgba(255, 255, 255, 0.5);*/
	border: 0px;
	border-radius: 2px;
	padding-top: 2px;
    padding-bottom: 2px;
}
QScrollBar{
    width: 10px;
    border: 0px;
    border-radius: 5px;
	padding: 2px;
}
QScrollBar::handle{
	width: 4px;
	border-radius: 2px;
	/*background-color: #84ACD9;*/
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar::handle:hover{
	/*background-color: #75ACEB;*/
	background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar:vertical {
    border: none;          /* 去除滚动条整体边框 */
    background: transparent;  /* 背景透明（可选） */
}
QScrollArea .QWidget{
	border: none;
}
QScrollArea{
	padding-top: 5px;
	padding-bottom: 5px;
}
/*垂直滚动条样式*/

/*推荐页面通用样式*/
[recommend]{
	font-weight: 700;
	font-size: 30px;
}
[day]{
	font-size: 18px;
}
/*推荐页面通用样式*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>每日推荐音乐</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="recommend" stdset="0">
          <string>-1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="day_lab">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>2 / 11</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="day" stdset="0">
          <string>-1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="recommend_music_scroll_area">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>340</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QVBoxLayout" name="recommend_music_vertical_layout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
